tsconfig.json

```diff
@@ -17,12 +18,15 @@
         "importHelpers": true,
         "rootDir": "src",
         "strictPropertyInitialization": false,
-        "target": "es5"
+        "target": "es2015",
+        "typeRoots": ["node_modules/@types"]
     },
-    "files": [
-        "src/index.ts"
-    ],
-    "exclude": [
-        "node_modules"
-    ]
+    "angularCompilerOptions": {
```

Comment:
Looks like this is the stuff necessary for Angular 9/Ivy

Reply:
Correct.